ports/mimxrt/boards/MIMXRT1170_EVK/qspi_hyper_flash_config.c

```diff
@@ -0,0 +1,186 @@
+/*
```

Comment:
Why is this file needed, because I see a common file `hal/qspi_hyper_flash_config.c` that I would think can be reused for this MCU?  If this code has to be different for the RT1170, then can this file be moved to `hal/qspi_hyper_flash_config_MIMXRT1170.c` or something like that?

Reply:
Actually this file is not needed any more, because replaced by the file in hal. So I remove it. Thank you for noticing. This branch was dangling around since Sep 2021, and since then the main mimxrt branch has been cleaned.